draft-ietf-sedate-datetime-extended.md

```diff
@@ -190,134 +190,60 @@ timestamp extension suffix and defines such a format that extends
 The format should allow implementations to specify additional
 important information in addition to the bare timestamp.
 This is done by defining *tags*, each with a *key* and
-a *value* separated by an equals sign, and
-allowing implementations to include an informative
+a *value* separated by an equals sign.
+The key of a tag can be split by including a hyphen/minus sign "`-`".
```

Comment:
is this also multiple possible components, or just two components?

Reply:
My intent was two components; the syntax this inherited also currently only allows two.
That is certainly not yet a decision, just the way it is being proposed now.

Reply:
Might be worth making that explicit here, maybe:

```
The key of a tag can be split into a namespace and subkey, by including a single hypen/minus sign
```

(substitute "subkey" with a better descriptor of that part if you have a better name for it - I was thinking "key name" but it's pretty ambiguous)

Reply:
I expanded it a bit:

```
The key of a tag can be split into two parts by including a
hyphen/minus sign "`-`"; the first part (including the "`-`") can then
be used as a namespace.
```